pymongo/encryption.py

```diff
@@ -1021,6 +1021,19 @@ def rewrap_many_data_key(
         :Returns:
           A :class:`RewrapManyDataKeyResult`.
 
+        This method allows you to re-encrypt all of your data-keys with a new CMK, or master key.
+        Note that this does *not* require re-encrypting any of the data in your encrypted collections,
+        but rather refreshes the key that protects the keys that encrypt the data::
+
+            client_encryption.rewrap_many_data_key(
```

Comment:
Can we wrap this in a code directive, so it gets rendered and prettified?

Reply:
Done.